sdk/communication/communication-phone-numbers/src/utils/helpers.ts

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+export const generateGUID = (): string => {
```

Comment:
Let's not make up our own definition of GUID, please use `randomUUID` from azure core util